gnocchi/rest/api.py

```diff
@@ -2116,10 +2116,14 @@ def get(details=True):
             report_dict['metricd']['processors'] = [
                 member.decode() for member in members
             ]
-            report_dict['metricd']['statistics'] = {
-                member.decode(): cap.get()
-                for member, cap in six.moves.zip(members, caps)
-            }
+            members_data = {}
+            for member, cap in six.moves.zip(members, caps):
```

Comment:
You can stop using `six` here.

Reply:
Considering there is a lot of pending fixes, I would like that we merge fixes, do a new release for the 4.3.x series (which would still include python2 support even though we don't explicitly test it now) then prepare for 4.4 or 5.0 (better?) and then drop all compatibility (six usage) etc.

Reply:
Feedback on how to test this change is appreciated.